core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -392,7 +393,7 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
       }
 
       results.map { responses => RequestComplete(StatusCodes.OK, responses) }
-    }
+    }, TransactionIsolation.ReadCommitted)
```

Comment:
be more okay with reading possibly-stale data from the workflows table when other people are writing to it. not super important that we have the TOTALLY LATEST last changed date, especially if workflows are being launched constantly